packages/natds-themes/bin/colors-to-code-output.json

```diff
@@ -0,0 +1,3796 @@
+{
```

Comment:
Esse arquivo vem de onde?

Reply:
de um plugin do figma chamado colors-to-code
https://www.figma.com/community/plugin/857997781988109369